test/parallel/test-eslint-inspector-check.js

```diff
@@ -12,12 +12,18 @@ const message = 'Please add a skipIfInspectorDisabled() call to allow this ' +
 new RuleTester().run('inspector-check', rule, {
   valid: [
     'foo;',
-    'common.skipIfInspectorDisabled(); require("inspector");'
+    'common.skipIfInspectorDisabled(); require("inspector");',
+    'require("common");\n' +
+    'common.skipIfInspectorDisabled();\n' +
+    'require("inspector");'
   ],
   invalid: [
     {
-      code: 'require("inspector")',
-      errors: [{ message }]
+      code: 'require("inspector");',
+      errors: [{ message }],
+      output: 'require("common");\n' +
+              'common.skipIfInspectorDisabled();\n' +
+              'require("inspector");'
```

Comment:
This is very certain not the correct output as `common` was not imported in the checked code.